### PR TITLE
Code Quality: Made private UserSettingsService property into a readonly field

### PR DESCRIPTION
**Pull request by QuaintMako**
**Resolved / Related Issues**
- Related to #11008

**What has been done**
- Replaced `private IUserSettingsService UserSettingsService { get; } = Ioc.Default.GetRequiredService<IUserSettingsService>();` by `private readonly IUserSettingsService userSettingsService = Ioc.Default.GetRequiredService<IUserSettingsService>();`
- Inverted one `if`

**Validation**
How did you test these changes?
- [x] Built and ran the app.

### CODE REVIEW

**Review by hecksmosis — APPROVED** [review]
LGTM

File: src/Files.App/DataModels/SidebarPinnedModel.cs [hunk 1]
```diff
@@ -277,12 +277,12 @@ private void AddLocationItemToSidebar(LocationItem locationItem)
...
+			if (!userSettingsService.PreferencesSettingsService.ShowFavoritesSection)
+				return;
+
+			foreach (string path in FavoriteItems)
 			{
-				foreach (string path in FavoriteItems)
-				{
-					await AddItemToSidebarAsync(path);
-				}
+				await AddItemToSidebarAsync(path);
```
Comment (ferrariofilippo) [inline src/Files.App/DataModels/SidebarPinnedModel.cs:285]: I think you can remove braces

File: src/Files.App/Filesystem/Search/FolderSearch.cs [hunk 2]
```diff
@@ -273,7 +273,7 @@ private async Task AddItemsAsync(string folder, IList<ListedItem> results, Cance
 					hiddenOnlyFromWin32 = (results.Count != 0);
 				}
 
-				if (!IsAQSQuery && (!hiddenOnlyFromWin32 || UserSettingsService.FoldersSettingsService.ShowHiddenItems))
+				if (!IsAQSQuery && (!hiddenOnlyFromWin32 || userSettingsService.FoldersSettingsService.ShowHiddenItems))
 				{
 					await SearchWithWin32Async(folder, hiddenOnlyFromWin32, UsedMaxItemCount - (uint)results.Count, results, token);
 				}
```
Comment (ferrariofilippo) [inline src/Files.App/Filesystem/Search/FolderSearch.cs:279]: Same as above

File: src/Files.App/UserControls/FilePreviews/MediaPreview.xaml.cs [hunk 3]
```diff
@@ -26,16 +26,16 @@ public MediaPreview(MediaPreviewViewModel model)
...
 		}
 
 		private void MediaPlayer_VolumeChanged(MediaPlayer sender, object args)
 		{
-			if (sender.Volume != UserSettingsService.PreviewPaneSettingsService.MediaVolume)
+			if (sender.Volume != userSettingsService.PreviewPaneSettingsService.MediaVolume)
 			{
-				UserSettingsService.PreviewPaneSettingsService.MediaVolume = sender.Volume;
+				userSettingsService.PreviewPaneSettingsService.MediaVolume = sender.Volume;
 			}
```
Comment (ferrariofilippo) [inline src/Files.App/UserControls/FilePreviews/MediaPreview.xaml.cs:39]: As above

File: src/Files.App/ViewModels/MainPageViewModel.cs [hunk 4]
```diff
@@ -346,49 +346,49 @@ public async void OnNavigatedTo(NavigationEventArgs e)
...
 								await AddNewTabByParam(tabArgs.InitialPageType, tabArgs.NavigationArg);
 							}
 						}
 
-						if (!UserSettingsService.PreferencesSettingsService.ContinueLastSessionOnStartUp)
+						if (!userSettingsService.PreferencesSettingsService.ContinueLastSessionOnStartUp)
 						{
-							UserSettingsService.PreferencesSettingsService.LastSessionTabList = null;
+							userSettingsService.PreferencesSettingsService.LastSessionTabList = null;
 						}
```
Comment (ferrariofilippo) [inline src/Files.App/ViewModels/MainPageViewModel.cs:374]: As above

File: src/Files.App/ViewModels/SidebarViewModel.cs [hunk 5]
```diff
@@ -532,37 +532,37 @@ private void UserSettingsService_OnSettingChangedEvent(object sender, SettingCha
...
 			switch (e.SettingName)
 			{
-				case nameof(UserSettingsService.AppearanceSettingsService.IsSidebarOpen):
-					if (UserSettingsService.AppearanceSettingsService.IsSidebarOpen != IsSidebarOpen)
+				case nameof(userSettingsService.AppearanceSettingsService.IsSidebarOpen):
+					if (userSettingsService.AppearanceSettingsService.IsSidebarOpen != IsSidebarOpen)
 					{
 						OnPropertyChanged(nameof(IsSidebarOpen));
 					}
 					break;
```
Comment (ferrariofilippo) [inline src/Files.App/ViewModels/SidebarViewModel.cs:540]: As above

File: src/Files.App/Views/ColumnShellPage.xaml.cs [hunk 6]
```diff
@@ -364,7 +364,7 @@ private async void ColumnShellPage_QuerySubmitted(ISearchBox sender, SearchBoxQu
 			}
 			else if (e.ChosenSuggestion is null && !string.IsNullOrWhiteSpace(sender.Query))
 			{
-				SubmitSearch(sender.Query, UserSettingsService.PreferencesSettingsService.SearchUnindexedItems);
+				SubmitSearch(sender.Query, userSettingsService.PreferencesSettingsService.SearchUnindexedItems);
 			}
```
Comment (ferrariofilippo) [inline src/Files.App/Views/ColumnShellPage.xaml.cs:368]: As above

File: src/Files.App/Views/PaneHolderPage.xaml.cs [hunk 7]
```diff
@@ -298,14 +298,14 @@ private void KeyboardAccelerator_Invoked(KeyboardAccelerator sender, KeyboardAcc
 			switch (c: ctrl, s: shift, m: menu, k: args.KeyboardAccelerator.Key)
 			{
 				case (true, true, false, VirtualKey.Left): // ctrl + shift + "<-" select left pane
-					if (UserSettingsService.PreferencesSettingsService.IsDualPaneEnabled)
+					if (userSettingsService.PreferencesSettingsService.IsDualPaneEnabled)
 					{
 						ActivePane = PaneLeft;
 					}
```
Comment (ferrariofilippo) [inline src/Files.App/Views/PaneHolderPage.xaml.cs:304]: As above

File: src/Files.App/Views/PaneHolderPage.xaml.cs [hunk 8]
```diff
@@ -298,14 +298,14 @@ private void KeyboardAccelerator_Invoked(KeyboardAccelerator sender, KeyboardAcc
...
 					break;
 
 				case (true, true, false, VirtualKey.Right): // ctrl + shift + "->" select right pane
-					if (UserSettingsService.PreferencesSettingsService.IsDualPaneEnabled)
+					if (userSettingsService.PreferencesSettingsService.IsDualPaneEnabled)
 					{
 						if (string.IsNullOrEmpty(NavParamsRight?.NavPath))
 						{
```
Comment (ferrariofilippo) [inline src/Files.App/Views/PaneHolderPage.xaml.cs:311]: As above

**Review by ferrariofilippo — COMMENTED** [review]
Overall, LGTM. I think you should remove braces of single-line ifs
Edit. I think that private non-static fields should be prefixed by '_'. See `2. Naming notation for variables` of [code style guidelines](https://files.community/docs/contributing/code-style)

**Comment by QuaintMako** [comment]
> Edit. I think that private non-static fields should be prefixed by '_'. See `2. Naming notation for variables` of [code style guidelines](https://files.community/docs/contributing/code-style)

From what I see, this rule is not enforced anymore, most of the code base doesn't use it. If we are to do it, that's gonna be a huge cost.

**Review by ferrariofilippo — DISMISSED** [review]
LGTM

**Comment by hecksmosis** [comment]
@QuaintMako Can you fix the compilation error?

**Comment by hecksmosis** [comment]
Line 135 of FoldersViewModel.cs

**Review by hecksmosis — DISMISSED** [review]
LGTM

**Review by ferrariofilippo — APPROVED** [review]
LGTM

**Review by hecksmosis — APPROVED** [review]
LGTM, but can you remove the FilesLauncher.exe, I don't think it's needed.

**Comment by yaira2** [comment]
@QuaintMako can you resolve the merge conflicts?

**Pull request closed without merging**